src/mbed_os_tools/detect/platform_database.py

```diff
@@ -178,10 +179,15 @@
         u"0853": u"NUCLEO_F301K8",
         u"0854": u"NUCLEO_L552ZE_Q",
         u"0855": u"DISCO_L562QE",
+        u"0858": u"DISCO_H750B",
+        u"0859": u"DISCO_H7B3I",
         u"0860": u"NUCLEO_H7A3ZI_Q",
+        u"0863": u"DISCO_L4P5G",
+        u"0865": u"NUCLEO_L4P5ZG",
         u"0866": u"NUCLEO_WL55JC",
         u"0871": u"NUCLEO_H723ZG",
         u"0872": u"NUCLEO_G0B1RE",
+        u"0875": u"DISCO_H735ZG",
```

Comment:
Proposal to match online database:
```suggestion
        u"0875": u"DISCO_H735G",
```

Reply:
Corrected